src/qibolab/pulses.py

```diff
@@ -21,6 +21,7 @@ class PulseType(Enum):
     READOUT = "ro"
     DRIVE = "qd"
     FLUX = "qf"
+    COUPLERFLUX = "cf"
```

Comment:
I know we already discussed, and you kept discussing with @stavros11, but I'm not up-to-date any longer.

Could you remind me why do we need an extra `PulseType` for the coupler flux?

From my perspective, a `Coupler` is a special kind of `Qubit`, and its flux than is the same of a qubit flux.
The only difference is that the pulse is sent to a coupler and not to a whatever other qubit.

Reply:
The pulses we have access in the drivers will only have `pulse.qubit` that may give a `Coupler` or `Qubit`. But you are giving it's name, so I had no way of telling which one it was without naming one of them is a specific way. Having the `COUPLERFLUX` allows me to know at all times what the `pulse.qubit` is without having a `pulse.coupler`.

Reply:
Ok, this is now covered by #605, and the way to tell the difference in that case will be `isinstance(Coupler)`. 

But in other cases, you should be able to query the platform, to ask the object associated to a given `QubitId`.